tentacle/src/runtime/async_runtime.rs

```diff
@@ -24,25 +31,47 @@ mod os {
     use tokio_util::compat::{Compat, FuturesAsyncWriteCompatExt};
 
     #[derive(Debug)]
-    pub struct TcpListener(AsyncListener);
+    pub struct TcpListener(
+        Receiver<io::Result<(AsyncStream, SocketAddr)>>,
+        SocketAddr,
+        Sender<()>,
+    );
 
     impl TcpListener {
+        fn new(listener: AsyncListener, local_addr: SocketAddr) -> TcpListener {
+            let (mut tx, rx) = channel(24);
+            let (tx_c, rx_c) = oneshot::channel::<()>();
+            let task = async move {
+                loop {
+                    let res = listener.accept().await;
+                    let _ignore = tx.send(res).await;
```

Comment:
Better log a debug message when send failed.

Reply:
There is only one possibility, that is, recv has been dropped, but in this case, the task will be turned off by select, so there is no condition to return error